backend/controller/ml_model/shared.py

```diff
@@ -23,6 +24,12 @@ class Source(shared.StrEnum):
   GOOGLE_ANALYTICS = 'GOOGLE_ANALYTICS'
 
 
+@dataclasses.dataclass
+class TimespanRange:
```

Comment:
nit:  Add a quick comment explaining what unit of measure the ints represent (ie, Unix epoch time).